packages/atomic/src/components/search/result-template-components/quickview-preview-bar/quickview-preview-bar.ts

```diff
@@ -0,0 +1,57 @@
+import {QuickviewWordHighlight} from '../quickview-word-highlight/quickview-word-highlight';
+
+const buildPreviewBar = (documentWriter: Document) => {
+  const bar = documentWriter.createElement('div');
+  bar.style.position = 'fixed';
+  bar.style.top = '0';
+  bar.style.right = '0';
+  bar.style.width = '15px';
+  bar.style.height = '100%';
```

Comment:
for accessibility, we can probably just add `aria-hidden="true"` to this, since screen reader users are already going to be able to navigate using the next/previous buttons.